server/models/Event.ts

```diff
@@ -24,6 +24,11 @@ export const EventSchema = new Schema({
         required: true,
         default: 0,
     },
+    groupSignUp: {
```

Comment:
ah ok, i will wait till you get this branch merged in and will delete the code I have :+1: